app/src/main/java/org/oppia/android/app/home/promotedlist/PromotedStoryViewModel.kt

```diff
@@ -56,8 +57,9 @@ class PromotedStoryViewModel(
   }
 
   fun clickOnStoryTile() {
+    val profileId = ProfileId.newBuilder().apply { internalId = internalProfileId }.build()
```

Comment:
Could a ProfileId be passed into this view model, instead, to avoid needing to create one here?

Reply:
Yeah updated, this too.